stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/listener/internal/HideChannelListenerState.kt

```diff
@@ -37,7 +33,7 @@ internal class HideChannelListenerImpl(
     ): Result<Unit> = validateCidWithResult(Pair(channelType, channelId).toCid()).toUnitResult()
 
     override suspend fun onHideChannelRequest(channelType: String, channelId: String, clearHistory: Boolean) {
-        logic.channel(channelType, channelId).setHidden(true)
+        logic.channel(channelType, channelId).stateLogic().setHidden(true)
```

Comment:
Can we change `setHidden` method name? I think it's misleading because it either hides or shows the channel based on the flag. Maybe it would be better to call it `changeHiddenValue`?

Reply:
I changed to toggleHidden